app/views/car_lot_vehicles/index.html.erb

```diff
@@ -1,5 +1,6 @@
 <h1>Vehicles</h1>
 <a href="/car_lots/<%= @car_lot.id %>/vehicles/new">create new vehicle</a>
+<a href="/car_lots/<%= @car_lot.id %>/vehicles">alphabetical order</a>
```

Comment:
Here is where we can pass down some info saying that the user clicked the link. Look into query params.